reference/user_defined_functions.rst

```diff
@@ -32,6 +32,8 @@ Arguments
 **distribution_method:** (Optional) The method according to which the table is
 to be distributed. Permissible values are append or hash, and defaults to 'hash'.
 
+**colocate_with:** (Optional) include current table in the co-location group of another table. By default tables distributed by columns of the same type and with the same shard count are co-located. Possible values are :code:`default`, :code:`none` to start a new co-location group, or the name of another table to co-locate with that table.  (See :ref:`colocation_groups`.)
```

Comment:
Looks like its tables which share the same (a) shard-count, (b) replication-factor and (c) distribution-column type. Could we adjust to reflect replication-factor as well?